GameServerLib/Game.cs

```diff
@@ -128,6 +112,12 @@ public void Initialize(Address address, string blowfishKey, Config config)
             _pauseTimer.Elapsed += (sender, args) => PauseTimeLeft--;
             PauseTimeLeft = 30 * 60; // 30 minutes
 
+            _packetServer = new PacketServer();
```

Comment:
Passing ``PacketServer`` as argument (maybe in another PR)?

Reply:
Ill consider to where to move it in another PR, currently there is some problem that PacketServer need IMap object or something, don't remeber exactly, so there are circular dependencies.